docs/data_sources/wip_future.rst

```diff
@@ -45,57 +44,55 @@ explaining why it would be useful.
 EIA Water Usage
 ^^^^^^^^^^^^^^^
 
-`EIA Water <https://www.eia.gov/electricity/data/water/>`_ records water use by
-thermal generating stations in the US.
+`EIA Water <https://www.eia.gov/electricity/data/water/>`_ records water use by thermal
+generating stations in the US.
 
 .. _data-ferc714:
 
 FERC Form 714
 ^^^^^^^^^^^^^
 
-`FERC Form 714 <https://www.ferc.gov/docs-filing/forms/form-714/data.asp>`_
-includes hourly loads, reported by load balancing authorities annually. This is
-a modestly sized dataset, in the 100s of MB, distributed as Microsoft Excel
-spreadsheets.
+`FERC Form 714 <https://www.ferc.gov/docs-filing/forms/form-714/data.asp>`_ includes
```

Comment:
maybe this one (and all the other sloan "new data" projects) should be pulled into the `Work in Progress` section?

Reply:
ok! I can just make one section with WIP